dockerfiles/ci-unified/Dockerfile

```diff
@@ -281,6 +275,18 @@ RUN	cargo install --git https://github.com/paritytech/ink-waterfall.git csv-comp
 # parity-scale-codec
 RUN	cargo +nightly-${RUST_NIGHTLY_VERSION} install grcov rust-covfix xargo dylint-link
 
+# Install rve-nightly-2023-04-05
+
+RUN wget https://github.com/paritytech/rustc-rv32e-toolchain/releases/download/rust-rve-nightly-2023-04-05-x86_64-unknown-linux-gnu/rust-rve-nightly-2023-04-05-x86_64-unknown-linux-gnu.tar.zst && \
+    tar --zstd -xf rust-rve-nightly-2023-04-05-x86_64-unknown-linux-gnu.tar.zst && \
+    mv tmp/rve-nightly /usr/local/rustup/toolchains && \
```

Comment:
It would be safer to use `/tmp` instead of `tmp`

Reply:
it's ./tmp here that is extracted from the tarball